client/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, InputHTMLAttributes, ChangeEvent } from 'react'
+import classnames from 'classnames'
+
+import './Radio.scss'
+
+export interface RadioProps extends InputHTMLAttributes<HTMLInputElement> {
+  label: string
+  checked: boolean
+  value: string
```

Comment:
Cant we accept any other type than string ?

Reply:
No.